packages/dnb-design-system-portal/src/shared/menu/PortalToolsMenu.tsx

```diff
@@ -71,6 +71,12 @@ export default function PortalToolsMenu({
           <Space top>
             <ChangeLocale />
           </Space>
+          <P size="small" top="x-small">
+            Read more about{' '}
+            <Anchor href="/uilib/usage/customisation/localization/">
+              localization
+            </Anchor>{' '}
```

Comment:
```suggestion
            </Anchor>
```
We probably don't need this, do we?